static/app/views/organizationStats/index.tsx

```diff
@@ -394,22 +394,25 @@ export class OrganizationStats extends Component<Props> {
                   />
                 </ErrorBoundary>
               </div>
-              {!isSingleProject && (
-                <ErrorBoundary mini>
-                  <UsageStatsProjects
-                    organization={organization}
-                    dataCategory={this.dataCategory}
-                    dataCategoryName={this.dataCategoryName}
-                    projectIds={this.projectIds}
-                    dataDatetime={this.dataDatetime}
-                    tableSort={this.tableSort}
-                    tableQuery={this.tableQuery}
-                    tableCursor={this.tableCursor}
-                    handleChangeState={this.setStateOnUrl}
-                    getNextLocations={this.getNextLocations}
-                  />
-                </ErrorBoundary>
-              )}
+              <ErrorBoundary mini>
+                {isSingleProject && (
```

Comment:
is this supposed to be `!isSingleProject`?

Reply:
Nope, it's intentional -- we want to show that heading only for 'single project view' because otherwise it'll only show a table view of the selected projects.

If you select 'All Projects' in the header, we don't need the heading here.
If you select 'projA, projB, projC' then the table only has those three entries
if you select just projA, we want the table to show 'All Projects' data (to fix the regression)